tests/test_core.py

```diff
@@ -181,16 +181,20 @@ def decorated(self):
 # run them with the JS code though.
 def also_with_impure_standalone_wasm(func):
   def decorated(self):
-    func(self)
+    func(self, shared.JS_ENGINES)
     # Standalone mode is only supported in the wasm backend, and not in all
     # modes there.
     if can_do_standalone(self):
       print('standalone (impure; no wasm runtimes)')
       self.set_setting('STANDALONE_WASM', 1)
+      # we will not legalize the JS ffi interface, so we must use BigInt
+      # support in order for JS to have a chance to run this without trapping
+      # when it sees an i64 on the ffi.
+      self.set_setting('WASM_BIGINT', 1)
       wasm_engines = shared.WASM_ENGINES
       try:
         shared.WASM_ENGINES = []
-        func(self)
+        func(self, [NODE_JS + ['--experimental-wasm-bigint']])
```

Comment:
I wonder if there is a nicer way to do this such as setting something on the instance like `self.js_engine? , but for now we can land this using this method.

Reply:
I was wondering that too. Another way is to set and unset `shared.JS_ENGINES` like the above code does for `WASM_ENGINES`?